components/tls/tls-mpc/src/msg.rs

```diff
@@ -1,7 +1,11 @@
+//! Contains message types for communication between prover and notary
+
 use mpz_core::hash::Hash;
 use serde::{Deserialize, Serialize};
 use tls_core::msgs::enums::ContentType;
 
+/// An enum for different message types on the TLS level
```

Comment:
let's call them "record types" instead of "message types" so that the word "message" could be reserved for protocol messages